src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java

```diff
@@ -230,6 +247,43 @@ public boolean getDefaultFormProcessing() {
         };
     }
 
+    public boolean addOtherSRS(ResourceInfo resourceInfo) {
```

Comment:
this function is used to read other srs and update metadata map of FeautreTypeInfo, during Resource creation/modification on UI